[index.md] rebrand Wootric to InMoment (#1940)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/wootric/index.md

---
title: InMoment (formerly Wootric)
rewrite: true
---

> [InMoment (formerly Wootric)](https://wootric.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is modern customer feedback management software. Easily incorporate Voice of Customer data in your decision-making processes and marketing campaigns, and trigger customer follow-up when you send survey responses to your CRM, marketing platform or data warehouses automatically.

Reviewer: ```suggestion
[InMoment](https://wootric.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) (formerly Wootric) is modern customer feedback management software. You can incorporate Voice of Customer data in your decision-making processes and marketing campaigns, and trigger customer follow-up when you send survey responses to your CRM, marketing platform or data warehouses automatically.
```

> This source is maintained by InMoment. For any issues with the source, [contact the InMoment support team](mailto:support@wootric.com).

Reviewer: ```suggestion
InMoment maintains this source. For any issues, [contact the InMoment support team](mailto:support@wootric.com).
```

## Getting Started

> 1. From your Segment UI’s Sources page click on “Add Source”.

Reviewer: ```suggestion
1. In Segment go to **Connections** and click **Add Source**.
```

> 2. Search for "InMoment" within the Sources Catalog and confirm by clicking "Connect".

Reviewer: ```suggestion
2. Search for **InMoment** in the Source Catalog and confirm by clicking **Connect**.
```

> 3. Give the Source a nickname and follow the set up flow to "Add Source". The nickname will be used to designate the source in the Segment interface, and Segment will create a related schema name. The schema name is the namespace you'll be querying against in your warehouse. The nickname can be whatever you like, but we recommend sticking to something that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).

Reviewer: ```suggestion
3. Give the Source a nickname and click **Add Source**. The nickname is used to designate the source in the Segment interface, and Segment creates a related schema name. The schema name is the namespace you query against in your warehouse. The nickname can be whatever you like, but Segment recommends using a name that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
```

> 4. Copy the Write key from the Segment UI and log in to your InMoment account - navigate to Settings > Integrations > Send InMoment Data > Send survey responses back to Segment and paste the key to connect.

Reviewer: ```suggestion
4. Go to the **Settings > API Keys**. 
5. Copy the **Write Key** and log in to your InMoment account. 
6. Navigate to **Settings > Integrations > Send InMoment Data > Send survey responses back to Segment** and paste the write key to connect.
```

> Log into your downstream tools and check to see that your events are populating and they contains all the properties you expect. If all your events and properties are not showing up, refer to the Destination docs for troubleshooting.

Reviewer: ```suggestion
Log into your downstream tools and check to see that your events are populating and they contain all the properties you expect. If all your events and properties are not showing up, refer to the Destination docs for troubleshooting.
```
